src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -45,13 +48,13 @@
 import java.util.logging.Logger;
 import java.io.File;
 import javax.annotation.Nonnull;
+
```

Comment:
nit: Not a big deal, but try to avoid miscellaneous whitespace-only changes unless that is all your PR is changing.